vllm/model_executor/models/deepseek.py

```diff
@@ -0,0 +1,454 @@
+# coding=utf-8
+# Adapted from
+# https://github.com/huggingface/transformers/blob/v4.28.0/src/transformers/models/llama/modeling_llama.py
+# Copyright 2023 The vLLM team.
+# Copyright 2022 EleutherAI and the HuggingFace Inc. team. All rights reserved.
```

Comment:
A quick question: Is this the correct copyright header for deepseek models? I found
```
# Copyright 2023 DeepSeek-AI and The HuggingFace Inc. team. All rights reserved.
```
in https://huggingface.co/deepseek-ai/deepseek-moe-16b-chat/blob/cc01c87767bd905af4cb364693fd107014694ab9/modeling_deepseek.py#L2